HEDremap.tsv

```diff
@@ -1 +1,24 @@
- 
\ No newline at end of file
+Event/Duration/#	(Event, Attribute/Duration/#)
+Event/ID/#			(Event, Attribute/ID/Value/#)
+Event/Group ID/#	(Event, Attribute/ID/Type/Group, Attribute/ID/Value/#)
+Event/Category/Experimental control/Sequence/Permutation ID/#	(Event/Category/Experimental control/Sequence, Attribute/ID/Value/#, Attibute/ID/Type/Sequence)
+Item/ID/#	(Item, Attribute/ID/Value/#)
+Item/ID/Local/#	(Item, Attribute/ID/Type/Local, Attribute/ID/Value/#)
+Item/ID/Group ID/#	(Item, Attribute/ID/Type/Group, Attribute/ID/Value/#)
+Participant/ID/#	(Participant, Attribute/ID/Type/Participant, Attribute/ID/Value/#)
+Attribute/State ID/#	(Attribute/ID/Value/#, Attribute/ID/Type/State)
+Attribute/Location/Reference frame/Relative to participant/Participant ID/#	(Attribute/Location/Reference frame/Relative to participant, Attribute/ID/Type/Participant, Atribute/ID/Value/#)
+Sensory presentation/Visual/Rendering type/Screen/View port/ID/#	(Sensory presentation/Visual/Rendering type/Screen/View port, Attribute/ID/Value/#)
+Attribute/Object side/Reference object ID/#	(Attribute/Object side, Attribute/ID/Type/Reference object, Attribute/ID/Value/#)
+Experiment context/#	(Experiment context, Attribute/ID/Value/#)
+Event/Category/Technical error/#	(Event/Category/Technical error, Attribute/ID/Value/#)
+Event/Category/Partipant failure/#	(Event/Category/Partipant failure, Attribute/ID/Value/#)
+Event/Category/Environmental/#	(Event/Category/Environmental, Attribute/ID/Value/#)
+Event/Category/Experimental procedure/#	(Event/Category/Experimental procedure, Attribute/ID/Value/#)
+Event/Category/Incidental/#	(Event/Category/Incidental, Attribute/ID/Value/#)
+Event/Category/Miscellaneous/#	(Event/Category/Miscellaneous, Attribute/ID/Value/#)
+Event/Category/Experiment control/Sequence/Block/#	(Event/Category/Experiment control/Sequence/Block, Attribute/ID/Value/#)
+Event/Category/Experiment control/Sequence/Trial/#	(Event/Category/Experiment control/Sequence/Trial, Attribute/ID/Value/#)
+Event/Category/Experiment control/Task/#	(Event/Category/Experiment control/Task, Attribute/ID/Value/#)
+Event/Category/Experiment control/Synchronization/Tag	(Event/Category/Experiment control/Synchronization, Attribute/ID/Type/Synchronization)
```

Comment:
What does it mean to use this tag without specifying ID value?

Reply:
My thinking was that some tags could just  like Attribute/ID/Type/Synchronization could be used without a specific ID value to just mark synchronization points or place other markers.  I was worried about parsing, but I think the following would work and maybe would be better:

**Attribute**  
   * ID   
      *  #  
      *  Group  
         * #  
      *  Local  
         * #  
       * Participant  
          * #  
       * Permutation  
          * #  
       *  State
           * #

Should I make the change?

Reply:
Is Event/Category/Experiment control/Synchronization itself sufficient to mark synchronization points? I think it's unintuitive to use ID tag without value to mark a type of event, as "ID" implies a unique identifier.

Reply:
I think we should not have mix IDs and special keywords like "Group", "Local" etc as suggested above at the exact same level (all directly under Attribute/ID) since then IDs cannot be "Group", "Local" etc and this is a problem: the users would have to maintain a list of special keywords that cannot be used as IDs, and the list may change across versions.

I also second "I think it's unintuitive to use ID tag without value to mark a type of event, as "ID" implies a unique identifier."

My suggestion: either (1) we use parenthesis to make IDs context-dependent using a different attribute, e.g. "Attribute/ID Type"

Event/Category/Experiment control/Synchronization/Tag/# -> (Event/Category/Experiment control/Synchronization, Attribute/ID/#, Attribute/ID Type/Synchronization)

OR

(2) we use a keyword like 'value' to distinguish the value of an ID from its type:

Event/Category/Experiment control/Synchronization/Tag/# -> (Event/Category/Experiment control/Synchronization, Attribute/Group/Value/#)

What we need to have in mind is that an event may have multiple ID types, so the first way of doing, option number 1, this may get longer with nested parentheses, like:

(Item/.., (Attribute/ID/#, Attribute/ID Type/Local), (Attribute/ID/#, Attribute/ID Type/Group))

If we want to avoid nested parentheses then option number 2 might be more desirable.